anydex/wallet/node/hosts.json

```diff
@@ -0,0 +1,17 @@
+{
+  "monero": [
```

Comment:
What decision process did you use to select these nodes? Are they somehow either approved by or directly provided by the cryptocurrency vendor? If not, would splitting them into different categories based on _trustworthiness_ be sensible?

Reply:
I agree, that's definitely a good idea. Right now we're only working with hosts functioning as Monero hosts, but as we add hosts for other cryptocurrencies it is worth looking at.